node/src/components/fetcher/tests.rs

```diff
@@ -7,13 +7,14 @@ use casper_node_macros::reactor;
 use futures::FutureExt;
 use tempfile::TempDir;
 use thiserror::Error;
-use tokio::time;
 
 use casper_types::ProtocolVersion;
 
 use super::*;
 use crate::{
-    components::{deploy_acceptor, in_memory_network::NetworkController, storage},
+    components::{
+        deploy_acceptor, in_memory_network::NetworkController, storage, tests_common::advance_time,
```

Comment:
Our convention is to not fully import functions, so this should be just `tests_common` and will become `tests_common::advance_time(time_to_advance).await;` at the callsite below.